.circleci/config.yml

```diff
@@ -413,7 +413,7 @@ jobs:
   insync-checkin:
     machine:
       image: ubuntu-1604:201903-01
-      docker_layer_caching: true
+      #docker_layer_caching: true
```

Comment:
Is this intentional?

Reply:
yeah, stupid bug with circleci's layer caching. will restore at some point after landing (when the outdated layers get evicted)